### ISSUE TITLE
Some issues while searching in docs

### ISSUE THREAD

**Issue by rafaelss95** [angular/angular#18872]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Feature request
[X ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
</code></pre>

## Current behavior
Some cases:

Searching by `accessor` or `valueaccessor` doesn't catch `ControlValueAccessor`;

Searching by `abstract` or `control`  doesn't catch `AbstractControl`;

In a nutshell, **some** "docs" can be found only if you type everything.

Others:
Strangely, or not, when you search `abstract` the `ControlValueAccessor` is caught... maybe because there's the word "abstracts" in the text description.

**Comment by petebacondarwin** [comment]
Typing  `abstract` does now find AbstractControl... but `accessor` still doesn't catch `ControlValueAccessor` :-(
